Makefile.am

```diff
@@ -115,6 +115,7 @@ roff_man_inst = man/GEOPM_CXX_MAN_Agent.3 \
                 # end
 
 roff_man_noinst = man/GEOPM_CXX_MAN_Comm.3 \
+                  man/GEOPM_CXX_MAN_GPUActivityAgent.3 \
```

Comment:
Any man pages we need to add should also be behind the beta flag.  This should removed, and the chapter 7 included in the ```roff_man_beta``` section.